commands/sheriff.js

```diff
@@ -1,10 +1,21 @@
+const Game = require('../src/game.js');
+
 module.exports = {
 	name: 'sheriff',
 	description: 'Create new sheriff game',
 	execute(client, message, args) {
-		console.log('message', message);
-		if(message.mentions.users.size < 5){
-            return message.channel.send('Five players are needed for a game of sheriff');
-        }
+		// console.log('message', message);
+		// if(message.mentions.users.size < 5){
+        //     return message.channel.send('Five players are needed for a game of sheriff');
+        // }
+
+		// Username#1318
+		let creator = `${message.author.username}#${message.author.discriminator}`;
+		// 07/16/2021 7:51:47 PM
+		let date = new Date(Date.now()).toLocaleDateString("en-US");
+		let time = new Date(Date.now()).toLocaleTimeString("en-US");
+		let createdOn = `${date} ${time}`;
```

Comment:
Same here. I have a bad experience with Date objects anyway. Would rather use [moment.js](https://momentjs.com/) for dates, but we also need to think about what we want for logging anyway.